ci/docker/Dockerfile

```diff
@@ -71,6 +72,9 @@ RUN chmod +x /usr/local/bin/tini \
   && mv /opt/selenium/chromedriver /opt/selenium/chromedriver-$CHROME_DRIVER_VERSION \
   && chmod 755 /opt/selenium/chromedriver-$CHROME_DRIVER_VERSION \
   && ln -fs /opt/selenium/chromedriver-$CHROME_DRIVER_VERSION /usr/bin/chromedriver \
+  && git clone https://github.com/masterzen/winrm-cli \
+  && (cd winrm-cli; GOPATH=$PWD PATH=$PATH:$PWD/bin make) \
+  && cp winrm-cli/bin/winrm /usr/local/bin/ \
```

Comment:
If the winrm binary is statically linked or owns all its dependencies, can we clean up the cloned `winrm` directory when we're done copying out the executable?

Reply:
It already is a few lines further down - `rm -rf /tmp/work`

Reply:
Is it safe to rm the workdir of the image we are building?

Reply:
Yes.